elasticsearch.md

```diff
@@ -779,6 +779,8 @@ In this example, we want to avoid that users belonging to group "press" can see
 
 **⚠️IMPORTANT** The `filter`and `fields` rules will only affect "read" requests, therefore "write" requests **will not match** because otherwise it would implicitly allow clients to "write" without the filtering restriction. For reference, this behaviour is identical to x-pack and search guard.
 
+**⚠️IMPORTANT** Beginning with version 1.27.0 all ROR internal requests from kibana will not match blocks containing `filter` and/or `fields` rules.
```

Comment:
maybe we should clarify that ROR internal API is used by ROR Kibana during login & config reloading?

Reply:
I think it's worth to show the example from tests you showed us in the Jira ticket. It very much gives the idea of the ramifications this change has.

Reply:
I added some additional info and changed examples of "block duplication strategy"